src/Dialect/ONNX/IndexExpr.hpp

```diff
@@ -677,6 +630,26 @@ class ArrayValueIndexCapture {
   bool hasDefault;
 };
 
+// Capture array of values given by attributes.
+class ArrayAttributeIndexCapture {
```

Comment:
I did not find the usage of this class.

Reply:
Good catch, it is used in my support for matmul. I am trying to separate the IndexExpr from the Krnl.matmul PRs, so that they may be easier to digest.